eng/common/scripts/Helpers/NativeCommand-Helpers.ps1

```diff
@@ -0,0 +1,58 @@
+# These functions are meant to be used within a cmdlet supporting `-WhatIf`, i.e.
```

Comment:
Another helper we have been in need of is the Retry function. Probably doesn't fit with the name of NativeCommand-Helpers but probably should go into a helper somewhere.